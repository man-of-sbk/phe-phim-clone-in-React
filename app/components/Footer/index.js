/**
 *
 * Footer
 *
 */

import React from 'react';
// import PropTypes from 'prop-types';
// import styled from 'styled-components';

function Footer() {
  return <div />;
}

Footer.propTypes = {};

export default Footer;
