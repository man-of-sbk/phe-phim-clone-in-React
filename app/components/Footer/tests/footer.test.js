import React from 'react';
import { render } from '@testing-library/react';

import Footer from '../index';

describe('<Footer/> Component', () => {
  it('Should correct snapshot', () => {
    const { container } = render(<Footer />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
