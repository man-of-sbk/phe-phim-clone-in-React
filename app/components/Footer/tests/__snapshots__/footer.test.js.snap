// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> Component Should correct snapshot 1`] = `
<div
  class="ant-row container Wrapper-dwom2b-0 bNAgYw"
>
  <div
    class="ant-col ant-col-xs-24 ant-col-lg-12"
  >
    <h4>
      Suncinema
    </h4>
    <div>
      2018 Thử nghiệm cộng đồng , chờ giấy phép MXH của Bộ TT & TT
    </div>
  </div>
  <div
    class="ant-col ant-col-xs-24 ant-col-lg-12"
  >
    <h4>
      Theo dõi chúng tôi
    </h4>
    <a
      class="social-link"
      href="/"
    >
      <img
        alt="phe phim youtube"
        src="https://phephim.vn/assets/images/youtube.png"
      />
    </a>
    <a
      class="social-link"
      href="/"
    >
      <img
        alt="phe phim facebook"
        src="https://phephim.vn/assets/images/facebook.svg"
      />
    </a>
    <a
      class="social-link"
      href="/"
    >
      <img
        alt="phe phim instagram"
        src="https://phephim.vn/assets/images/insta.svg"
      />
    </a>
  </div>
</div>
`;
